<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>respuesta_sii_cons</class>
 <widget class="QDialog" name="respuesta_sii_cons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RESPUESTA ENVÍO SII</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>EMISOR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Num/Serie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre/Razón</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Importe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descripción</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Base1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cuota1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Base2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cuota2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Base3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cuota3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Base exenta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cuadre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="xml_pushButton">
       <property name="text">
        <string>Ver &amp;XML</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/xml.jpeg</normaloff>:/iconos/graf/xml.jpeg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copiar_pushButton">
       <property name="text">
        <string>&amp;Copiar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptar_pushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>xml_pushButton</tabstop>
  <tabstop>copiar_pushButton</tabstop>
  <tabstop>aceptar_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>aceptar_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>respuesta_sii_cons</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
